freeipa/src/main/java/com/sequenceiq/freeipa/sync/FreeipaStatusInfoLogger.java

```diff
@@ -0,0 +1,36 @@
+package com.sequenceiq.freeipa.sync;
+
+import com.sequenceiq.freeipa.entity.InstanceMetaData;
+import com.sequenceiq.freeipa.entity.Stack;
+import com.sequenceiq.freeipa.service.stack.instance.InstanceMetaDataService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+public class FreeipaStatusInfoLogger {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FreeipaStatusInfoLogger.class);
+
+    @Inject
+    private InstanceMetaDataService instanceMetaDataService;
+
+    public void logFreeipaStatus(Stack stack, Set<InstanceMetaData> checkableInstances) {
+        String freeipaStatusInfo = String.format("freeipa %s is %s. ", stack.getResourceCrn(), stack.getStackStatus().getStatus());
```

Comment:
for this to work right I think you have to fetch stack or stack status from db or you would have to modify `StackStatusCheckerJob#updateStackStatus` to return the modified stack and use that.
You could use `StackStatusService#findFirstByStackIdOrderByCreatedDesc` to fetch the current status.
Also resource crn logging shouldn't be necessary, the MDCContext should have all the filed set.